steam/user.py

```diff
@@ -62,6 +62,7 @@ class _BaseUser(BaseUser):
 
     def __init__(self, state: ConnectionState, proto: UserProto):
         super().__init__(state, proto.friendid)
+        self.rich_presence = None  # ensure the attr exists
```

Comment:
maybe this should be `self.rich_presence = {}` just so it matches `dict` in typing ? Oups

Reply:
I think leaving it as None is better because it indicates it didn't come empty from steam?